geocoding/README.md

```diff
@@ -0,0 +1,6 @@
+# Steps to geocode -
+
+1. `pip install -U googlemaps`
+2. Change `xlsx_name = "ReportedTravel.xlsx"` with the name of your xlsx file or path to it.
+3. Enter your API key in the code.
+4. Run the script `python geocode.py`
```

Comment:
Parameterize the script to take API Key as a terminal input using [Fire](https://github.com/google/python-fire) or [Click](https://click.palletsprojects.com/en/7.x/)? or from OS ENV variable?

Reply:
Parameterized variables as arguments using fire.